torch/csrc/jit/runtime/interpreter.h

```diff
@@ -81,6 +81,7 @@ struct TORCH_API MobileCode : Code {
   explicit MobileCode(
       const std::shared_ptr<Graph>& graph,
       std::string function_name,
+      bool emit_default_input_instructions = false,
```

Comment:
Maybe the default value should be `True`. When we decide to enable FC for mobile, we can flip value used on line 72 of export_module.cpp to `False`. Eventually, this flag can be removed.